massa-models/src/composite.rs

```diff
@@ -6,45 +6,67 @@ use massa_signature::{PublicKey, Signature};
 use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 
+/// Status in which an operation can be (derived from the block status)
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub enum OperationSearchResultBlockStatus {
+    /// the block hasn't been processed by consensus yet
     Incoming,
+    /// the block waits for it's slot for further processing
     WaitingForSlot,
+    /// the block waits for dependencies for further processing
     WaitingForDependencies,
+    /// the block has been processed and is valid
     Active,
+    /// the block is discarded
     Discarded,
-    Stored,
 }
 
+/// Status in which an operation can be
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub enum OperationSearchResultStatus {
+    /// in pool
     Pending,
+    /// in a block, the block being in OperationSearchResultBlockStatus
```

Comment:
Use the link method : https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html